docs/api/events.md

```diff
@@ -0,0 +1,325 @@
+# `Event`
+An `Event` describes an occurrence within a given time frame, such as an exam, a meeting or an appointment. 
+
+## Properties
+* `id`: the unique identifier of this `Event`
+  - type: **integer**
+  - e.g. 38427
+
+* `title`: title of the `Event`
+  - type: **text**
+  - e.g. "DWA 1st Exam"
+
+* `description`: description of the `Event`
+  - type: **text**
+  - e.g. "First exam for the DWA-s1920v class. Students are free to bring a cheat sheet of only 1 page, written by hand."
+
+* `start_date`: starting date of the `Event`
+  - type: **date**
+  - e.g. 19/03/2020
+
+* `end_date`: ending date of the `Event`
+  - the `end_date` should be after `start_date`
+  - type: **date**
+  - e.g. 30/04/2020
+
+## Link Relations:
+An event representation:
+* *must* include a link to its context, using the `self` link relation
```

Comment:
@joao-pereira5 The context is supposed to be linked to using the `about` relation right?

Reply:
Cited from the IANA registry: `about` relation